docs/platform/porting/graphics-and-ui.md

```diff
@@ -662,22 +725,23 @@ The following table lists the available project Git repositories.
 
 | Project         | Repository                               | Description                              |
 | --------------- | ---------------------------------------- | ---------------------------------------- |
-| `libtpl-egl`    | `platform/core/uifw/libtpl-egl`          | Tizen Porting Layer for EGL&trade;       |
-| `libtbm`        | `platform/core/uifw/libtbm`              | Library for Tizen Buffer Manager |
-| `coregl`        | `platform/core/uifw/coregl`              | Injection layer of OpenGL&reg; ES / EGL&trade;    |
+| `libtpl-egl`    | `platform/core/uifw/libtpl-egl`          | Tizen Porting Layer for EGL       |
+| `libtbm`        | `platform/core/uifw/libtbm`              | Library for the Tizen Buffer Manager |
```

Comment:
While resolving the conflict, it seems this "the" is added back.
Would you please remove "the" in front of Tizen Buffer Manager?

```suggestion
| `libtbm`        | `platform/core/uifw/libtbm`              | Library for Tizen Buffer Manager |
```